.github/workflows/master-push.yml

```diff
@@ -0,0 +1,564 @@
+          with:
+            xcode-version: ${{ env.DOC_VERSION }}
+        - name: Prepare docs for packaging
+          run: sh -x build.sh release-package-docs
+        - name: Upload docs to artifacts library
+          uses: actions/upload-artifact@v3
+          with:
+            name: realm-docs
+            path: docs/realm-docs.zip
+  build-examples:
+      runs-on: macos-13
+      name: Package examples
+      needs: prepare
+      steps:
+          - name: Checkout
+            uses: actions/checkout@v4
+          - name: Prepare examples for packaging
+            run: sh -x build.sh release-package-examples
+          - name: Cache examples
+            uses: actions/cache/save@v3
+            with:
+              path: |
+                realm-examples.zip
+              key: realm-examples
+              enableCrossOsArchive: true
+  build-product: # Creates framework for each platform, xcode version, target and configuration
+      runs-on: macos-13
+      name: Package framework 
+      needs: prepare
+      strategy:
+        max-parallel: 20 # Blocks of 20, therefore if any of the build fails, we don't get a lot of XCode Clouds builds hanging, which are expensive.
+        matrix:
+          platform: ${{ fromJSON(needs.prepare.outputs.BUILD_PLATFORM_MATRIX) }}
+          target: [RealmSwift, Realm]
+          xcode-version: ${{ fromJSON(needs.prepare.outputs.XCODE_VERSIONS_MATRIX) }}
+          configuration: [release, static]
+          exclude:
+            - platform: osx
+              configuration: static
+            - platform: tvos
+              configuration: static
+            - platform: watchos
+              configuration: static
+            - platform: catalyst
+              configuration: static
+            - platform: tvossimulator
+              configuration: static
+            - platform: watchossimulator
+              configuration: static
+            - target: RealmSwift
+              configuration: static
+          include: 
+            - platform: visionos
+              target: Realm
+              xcode-version: 15.1
+              configuration: release
+            - platform: visionos
+              target: RealmSwift
+              xcode-version: 15.1
+              configuration: release
+            - platform: visionossimulator
+              target: Realm
+              xcode-version: 15.1
+              configuration: release
+            - platform: visionossimulator
+              target: RealmSwift
+              xcode-version: 15.1
+              configuration: release
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+        - name: Get Token
+          id: token
+          run: |
+            token=$(echo $(ruby ./scripts/xcode_cloud_helper.rb --issuer-id ${{ secrets.APPLE_STORE_CONNECT_ISSUER_ID }} --key-id ${{ secrets.APPLE_STORE_CONNECT_KEY_ID }} --pk "${{ secrets.APPLE_STORE_CONNECT_API_KEY }}" get-token))
```

Comment:
Is the extra `$(echo ...)` doing something complicated here? It should be just a no-op wrapper.